19-ggplot2-internals.Rmd

```diff
@@ -7,18 +7,30 @@
 - What the division of labor between `{ggplot2}` and `{grid}`?
 - What is the basic structure of/motivation for ggproto?
 
+<details>
+<summary>Libraries and Helper Functions</summary>
+
 ```{r 20-01, message=FALSE}
 library(ggplot2)
 library(ggtrace) # remotes::install_github("yjunechoe/ggtrace")
 library(purrr)
 library(dplyr)
+
+# source("utilities/workflows-inspect.R")
+# source("utilities/aliases.R")
+
+obj_byte <- function(x) {
+  scales::label_bytes()(as.numeric(object.size(x)))
+}
```

Comment:
The `obj_byte` function was made by a previous cohort member to measure the size of R objects (here: at various stages of the processing in producing and then printing a ggplot object)

Otherwise, I was trying to fix the `gg_inspect_vars` usage , so that's why I had loaded two two scripts from the `ggtrace` package.  This attempt didn't work, so I commented out the `source` lines to leave as breadcrumbs for a future cohort.